release-notes-7.0.0.md

```diff
@@ -0,0 +1,16 @@
+!not-ready-for-release!
+
+#### Version Number
+${version-number}
+
+#### New Features
+- None
+
+#### Breaking Changes
+- US915147: Liveness and readiness check support has been added to the `DocumentWorker` interface.
```

Comment:
This is really going to hurt.  It's one thing having breaking changes in components that don't have that many direct consumers (like `HealthReporter` in `caf-common`), but changing `DocumentWorker` in a breaking way will make life too difficult.